Makefile

```diff
@@ -963,17 +963,32 @@ bench-ci: bench
 lint-md-clean:
 	$(RM) -r tools/remark-cli/node_modules
 	$(RM) -r tools/remark-preset-lint-node/node_modules
+	$(RM) tools/.*mdlintstamp
 
 lint-md-build:
 	if [ ! -d tools/remark-cli/node_modules ]; then \
 		cd tools/remark-cli && ../../$(NODE) ../../$(NPM) install; fi
 	if [ ! -d tools/remark-preset-lint-node/node_modules ]; then \
 		cd tools/remark-preset-lint-node && ../../$(NODE) ../../$(NPM) install; fi
 
-lint-md: lint-md-build
-	@echo "Running Markdown linter..."
-	$(NODE) tools/remark-cli/cli.js -q -f \
-		./*.md doc src lib benchmark tools/doc/ tools/icu/
+LINT_MD_TARGETS = src lib benchmark tools/doc tools/icu
+LINT_MD_FILES := $(shell find $(LINT_MD_TARGETS) -type f \
+  -not -path '*node_modules*' -name '*.md') $(shell ls ./*.md)
```

Comment:
Is there a reason to shell out to `find` and `ls` instead of using `$(wildcard ...)`?

Reply:
I have tried to use `$(wildcard ..)`, but will need to filter out files inside `node_modules` and loop over the `LINT_MD_TARGETS`...not sure if there is a better way to write this?

Reply:
It's not super duper pretty but you can filter the results of `$(wildcard ...)` like this:
```make
LINT_MD_FILES := \
    $(foreach x,$(wildcard ...), \
        $(if $(findstring node_modules,$(x)),,$(x)))
```